index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>BJET Task</title>
+</head>
+<body>
+    <h1>BJET Task</h1>
+    <h2>BJET Task 2</h2>
+    <h3>BJET Task 3</h3>
```

Comment:
You did a great job 👍 . But still needs some changes. You added three titles with h1, h2, h3. All of them are on the same level. As per HTML Best Practice, at the same level, you should use the same title tag. 
For example :
```suggestion
    <h2>BJET Task</h1>
    <h2>BJET Task 2</h2>
    <h2>BJET Task 3</h3>
```

To learn more about it go to [this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#navigation).